runtime/common/src/weights/block_weights.rs

```diff
@@ -1,6 +1,7 @@
+
```

Comment:
Is this file using a different template than one we have? I don't see this extra line in the template, but it is what is causing the lint issue.

Reply:
`block_weights.rs` and `extrinsic_weights.rs` are generated by running:
`${BENCHMARK} overhead --execution=wasm --wasm-execution=compiled --weight-path=runtime/common/src/weights --chain=dev --warmup=10 --repeat=100 || exit_err`